README.adoc

```diff
@@ -418,10 +422,11 @@ Content for CityGML package
 
 ==== Defining tables
 
-.<<tab-P-another-C-{XMI_ID}>> -- Elements of &#8220;Another::AbstractAtomicTimeseries&#8221; (class)
+[arabic]
+.<<section-EAPK_9C96A88B_E98B_490b_8A9C_24AEDAC64293>> -- Elements of Another::AbstractAtomicTimeseries
```

Comment:
Remove `[arabic]` and add ` (class)` at the end of caption title.

Also, sometimes the package name (here as "Another") can contain spaces, so we should "double quote" the package name/class to differentiate 'Elements of "package name one::two"'.